### DISCUSSION TITLE
CYD2USB ALPHA 1

### DISCUSSION THREAD

**Discussion by GthiN89** [GthiN89/NucleusESP32#9] [General]
kinda works
pins are:
#define CYD_MOSI 21   // Master Out Slave In
#define CYD_MISO 35  // Master In Slave Out
#define CYD_SCLK 22  // Serial Clock
#define RFID_CS -1
#define RF24_CS 16

// gd0 17
// sck 22
// miso 35
// CKN 27
// mosi 21
// GD2 - 4

#define IRQ_PIN CCGDO2A
#define RFID_RST -1
#define RF24_CE  CCGDO0A

// Pin configuration for CC1101 nfc CC-27 NRF24 CC-16
#define CC1101_CS 27    // Chip Select
#define CC1101_MOSI CYD_MOSI   // Master Out Slave In
#define CC1101_MISO CYD_MISO  // Master In Slave Out
#define CC1101_SCLK CYD_SCLK  // Serial Clock

#define CCGDO0A 17      // GDO0-NFC IRQ, NF IRQ - input only
#define CCGDO2A 04      // GDO2 -rst - input output

#define CC1101_CCGDO0A CCGDO0A      // GDO0
#define CC1101_CCGDO2A CCGDO2A      // GDO2 -rst NF CE

<hr /><em>This discussion was created from the release <a href='https://github.com/GthiN89/NucleusESP32/releases/tag/'>CYD2USB ALPHA 1</a>.</em>

**Comment by OPAMINULFFyt** [comment]
Please stable release 🙏🙏🙏🙏

**Comment by GthiN89** [comment]
Im currently busy with porting of Momentum FW to  5 euro chinessse STM32
board, look around YT, hackaday tiktok etc.... (Yep, it does really work,
basically sub 20 euro flipper)
